app/src/main/java/org/mozilla/focus/Components.kt

```diff
@@ -71,9 +74,12 @@ class Components(
     private val clientOverride: Client? = null
 ) {
     val appStore: AppStore by lazy {
-        AppStore(AppState(
-            screen = determineInitialScreen(context)
-        ))
+        AppStore(
+            AppState(
+                screen = determineInitialScreen(context),
+                topSites = topSitesStorage.cachedTopSites
```

Comment:
nit: Since the app state is a global singleton, I assume topSites will always be an empty list at this point anyways? 🤔

Reply:
Doesn't this also mean we have two caches now? One in the AppStore and one in the top sites storage? Do we need both?

Reply:
Made AppStore our primary cache.